core/storage/base_model/gae_models.py

```diff
@@ -173,6 +193,14 @@ def get_export_policy(cls):
             'deleted': EXPORT_POLICY.NOT_APPLICABLE
         }
 
+    @classmethod
+    def get_export_policy_exceptions(cls):
```

Comment:
To be clear, you are saying that we should just rename this method? Instead of exceptions, indicate that this is just a mapping with a default setting?

Reply:
Discussed offline with @seanlip that we are going to rename this to get_renamed_key_names or something of that nature (I'll think more about the exact name).

Reply:
Add some code that renames the fields in Takeout according to what this method defines (and by default the export_data functions should export the field names).